ebu_tt_live/bindings/__init__.py

```diff
@@ -724,6 +724,8 @@ def _semantic_before_traversal(self, dataset, element_content=None, parent_bindi
     def _semantic_after_traversal(self, dataset, element_content=None, parent_binding=None):
         self._semantic_postprocess_timing(dataset=dataset, element_content=element_content)
         self._semantic_manage_timeline(dataset=dataset, element_content=element_content)
+        self._semantic_validate_region_extent(dataset=dataset)
```

Comment:
The semantic validation runs from the tt:element all the way down to the children element so, I've had to add validation to the whole ebuttd doc so that the dataset needed for the d_p_type can have the stuff it needs to be validated.